rules/linux/execution_python_tty_shell.toml

```
@@ -18,11 +20,14 @@ risk_score = 73
 rule_id = "d76b02ef-fc95-4001-9297-01cb7412232f"
 severity = "high"
 tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution"]
+timeline_id = "e70679c2-6cde-4510-9764-4823df18f7db"
+timeline_title = "Comprehensive Process Timeline"
 timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.category:process and event.type:(start or process_started) and process.name:python and
+event.category:process and event.type:(start or process_started) and 
+  process.name: (python or python3) and
```

might as well make it even more compatible

```suggestion
  process.name:python* and
```